src/main/java/cst438flights/controller/flightRestController.java

```diff
@@ -0,0 +1,48 @@
+package cst438flights.controller;
+
+import cst438flights.domain.Flight;
+import cst438flights.domain.FlightInfo;
+import cst438flights.service.FlightService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+public class flightRestController {
+
+    @Autowired
+    private FlightService flightService;
+
+    @GetMapping("/api/flights/{flight_ID}")
```

Comment:
I think this is left over from my original code base, its not pulling the JSON properly. I'll have to look into this.